_pages/how-to-submit.md

```diff
@@ -0,0 +1,213 @@
+
+### Domain specific
+
+This section is currently evolving, and it's OK if your package doesn't fit into a specific listed domain.
+
+### Community Partnerships
+
+Similar to the Domain specific section, our Community Partnerships are continuously growing and changing, and it's OK if your package isn't affiliated with one of the listed partners.
+
+### For all submissions
+
+This section is important, as it helps us more holistically evaluate your package and determine whether or not it's a good fit for pyOpenSci review. Each of the following sections should be addressed/answered:
+* *Explain how and why the package falls under the categories you indicated above.* Here's where you'll provide us with information that connects your package and its functionality to both its domain and technical capabilities.
+* *Who is the target audience and what are scientific applications of this package?* This helps us understand who will be using the package.
+* *Are there other Python packages that accomplish the same thing? If so, how does yours differ?* This helps us get a handle on other packages that fall within the same domain, as well as ensure that efforts aren't being duplicated.
+* *If you made a pre-submission enquiry, please paste the link to the corresponding issue, forum post, or other discussion, or `@tag` the editor you contacted.* This is an important section, as it links your pre-submission inquiry with your submission.
+
+### Technical checks
+
+It's important that your package meet each of the qualifications in this section. If you are missing one or more of these items, you can learn more about how to implement them in our [pyOpenSci Packaging Guide](https://www.pyopensci.org/python-package-guide/). Taken as a whole, these components represent best practices in Python packaging, and as such its important that they be present in all pyOpenSci-approved packages.
+
+### Publication Options
+
+One of the benefits of submitting your package to pyOpenSci for review is that once it's approved, it can be fast-tracked for publication with [JOSS](https://joss.theoj.org/). JOSS accepts pyOpenSci's review as theirs, and therefore you will not need to go through another full review. Instead, JOSS will only review your paper.md file.
+
+### Are you OK with Reviewers Submitting Issues and/or pull requests to your Repo Directly?
+
+You'll notice that the section on having reviewers submit requested changes as issues to your package repositories is already checked. You do have the option of un-checking it, but we consider the benefits of this to be that people can contribute to your review repos. Open issues, PRs, etc. all help you as a maintainer, while also giving you the opportunity to address any issues via PR links.
```

Comment:
Can you please re-read and reword a bit? id lead with the benefits / the why this section is there. Then tell them if they don't want issues and pr's they can uncheck it. as written it seems to combine these messages in a hard to digest way. 

we like to use the open source approach to github where ever we can. so linking issues and pr's to a review just makes the entire process more transparent in the spirit of our values. it also can help the maintianer directly if a pr is submitted!